test/test_sbol2_sbol3_direct.py

```diff
@@ -244,6 +253,82 @@ def test_seq_componentDefinition(self):
                 tmp2 = Path(tmpdir) / 'doc2_loop.xml'
                 doc2_loop.write(tmp2)
                 self.assertFalse(file_diff(str(tmp2), str(TEST_FILES / 'seq_componentDefinition.xml')))
+
+    def handle_2to3_conversion(self, test_filename, rubric_filename):
+        """Provides a re-usable test handler for converting SBOL2 to SBOL3 with different test files
+        test_filename: Name of an SBOL2 test file
+        rubric_filename: Name of an SBOL3 file with expected conversion
+    """
+        """Test ability to convert a simple part from SBOL2 to SBOL3"""
+
+        # Load an SBOL2 document and check its contents
+        doc2 = sbol2.Document()
+        doc2.read(TEST_FILES / test_filename)
+
+        # Convert to SBOL3 and check contents
+        doc3 = convert2to3(doc2, use_native_converter=True)
+        report = doc3.validate()
+        if len(report):
+            raise SBOLValidationError(report)
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            tmp3 = Path(tmpdir) / 'doc3.nt'
+            doc3.write(tmp3)
+            if file_diff(str(tmp3), str(TEST_FILES / rubric_filename)):
```

Comment:
Calling this `rubric_filename` seems odd to me. Maybe `expected_converted_file`?